src/datasource.js

```diff
@@ -119,8 +119,15 @@ function (angular, _, dateMath, moment) {
 
       }
       else if (target.queryType === 'Arithmetic') {
+          var expression = target.expression || '';
+          var queryLetters = Object.keys(targetsByRefId).filter(x => expression.indexOf(x + '[') !== -1);
```

Comment:
why are we filtering out arithmetic queries ?

Reply:
I don't really know why this was changed on the last PR, but apparently this code is the reason why it is not possible to make nested arithmetic grouped metrics.
As soon as I have some time I will take a look at this.

Reply:
Thank you

Reply:
@Gauravshah if the expression did not use some metric, then the metric is not considered here.

Reply:
is there any advantage to skip ? people can use `A.var1` as well. And I am not sure what else would they use to access